OxfordBodleian/Juel-Jensen/BDLaethe38.xml

```diff
@@ -42,11 +42,271 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                <persName ref="PRS12982Ananiqos"><roleName type="title">ʾabbā</roleName> ʾAnāniqos</persName>, 
+                                <persName ref="PRS12984TasfaMi"/>, 
+                                and <persName ref="PRS12988WaldaSe"><roleName type="title">ʾabbā</roleName> Walda Śǝllāse</persName>.
+                                According to the colophon, the book was written in the monastery of <placeName ref="LOC2297DabraL"/>, in <placeName ref="LOC5597Sawa"/>.
+                                Two short <foreign xml:lang="gez">salām</foreign>-hymns in honour of Ḥannā and ʾIyāqem are embedded in the colophon. 
+                                They are separated with chains of red and black dots or with black and red lines.
+                                </note>
+                        </colophon>
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                            <locus from="11ra" to="69rb"/>
+                            <title type="complete" ref="LIT1460Gadlah"/>
+                            <msItem xml:id="ms_i2.1">
+                                <locus from="11ra" to="12ra"/>
+                                <title type="complete" ref="LIT1460Gadlah#Prologue"/>
+                                <incipit xml:lang="gez">
+                                    <locus target="#11ra"/>
+                                    <hi rend="rubric">በስመ፡ እግዚአብሔር፡ አብ፡ ገባሬ፡</hi> ሰማያት፡ ወምድር፡ ህላዌሁ፡ ዘኢይ<hi rend="rubric">ትአመር፡ እምልበ፡ መዋቲ፡ 
+                                        ስውር፡</hi> ወዘኢይትከሀል፡ ይበልዎ፡ እምዝ፡ ለዝ፡ ውስተ፡ ዝ፡ ብጥንተ፡ ነገር። ዳዕሙ፡ ውእቱ፡ ህላዌሁ፡ የአምር።
+                                </incipit>
+                                <explicit xml:lang="gez">
+                                    <locus target="#12ra"/>
+                                    ሊተኒ፡ ያፈትወኒ፡ ከመ፡ እንግር፡ ዕበያ፡ ለይእቲ፡ ገነት፡ ወምድር። ምድርሂ፡ በእንተ፡ <hi rend="rubric">ሐና፡</hi> ወገነትሂ፡ በእንተ፡ እግዝእትነ፡ 
+                                    <hi rend="rubric">ማርያም፡</hi> ወፍሬሃኒ፡ በእንተ፡ ወልዳ፡ እግዚእነ፡ ወመድኃኒነ፡ <hi rend="rubric">ኢየሱስ፡ ክርስቶስ፡</hi> ሎቱ፡ ስብሐት፡ 
+                                    <add place="overstrike">ጸሎታ፡ ለቅድስት፡ <hi rend="rubric">ሐና፡</hi> ያድኅኖ፡ 
+                                        ለ<persName ref="PRS12983GabraMa" role="patron"><hi rend="rubric">ገብረ፡ ማርያም፡</hi></persName> <expan><abbr>ለዓ፡</abbr><ex>ለዓለመ፡</ex> 
+                                            <abbr>ዓ፡</abbr><ex>ዓለም፡</ex> <abbr>አ።</abbr><ex>አሜን።</ex></expan></add>
+                                </explicit>
+                            </msItem>
+                            <msItem xml:id="ms_i2.2">
+                                <locus from="12rb" to="26ra"/>
+                                <title type="complete" ref="LIT1460Gadlah#Monday"/>
+                                <incipit xml:lang="gez">
+                                    <locus from="12rb" to="12va"/>
```

Comment:
Is it correct to give from here on for all incipits and explicits a range "from xx to xx"?, not only the starting point?

Reply:
@thea-m